.circleci/config.yml

```diff
@@ -118,6 +118,14 @@ jobs:
       - store_artifacts:
           path: _out
 
+  call_marvin:
+    docker:
+      - image: circleci/golang:1.12.9
+    resource_class: medium
```

Comment:
why do we need any specific resource class here?

Reply:
Agree about the resource class its a simpe curl call no need for any specific compute demands